### ISSUE TITLE
Language filter (or, Language based repositories)

### ISSUE THREAD

**Issue by valerio-bozzolan** [btimofeev/instead-launcher-android#21]
Wow! There are a LOT of stories in russian!

Sadly I don't understand anything in this language so I was wandering if there is a way to introduce a language filter (e.g. English).

What do you think about? Thank you! :)

**Comment by btimofeev** [comment]
Hello. Thanks for the idea. Someday I will implement it, but I can not say how soon. 

Now you can find games in English through the search. Here are the names of the games in English:
- The Returning of the Quantum Cat
 - I.A.G. Alpha
 - Constellation
 - Mainland
 - Return of Red Hood
 - Crash Dive!
 - Semyon The Hamster 1
 - Am I a monster?
 - Kingdom of Grain
 - Mirror
 - Kayleth
 - Dark Crystal
 - The Archive

**Comment by valerio-bozzolan** [comment]
Thank you! Nice list. I will play some of them soon!

>  Someday I will implement it, but I can not say how soon.

No problem. Uhm. Can you indicate more or less where, in the source code, is the download part of the list? So maybe someone can help you with a pull request.

Thank you so much!

**Comment by btimofeev** [comment]
Updating the list of games is performed by this class https://github.com/btimofeev/instead-launcher-android/blob/master/app/src/main/kotlin/org/emunix/insteadlauncher/helpers/network/RepoUpdater.kt

But as far as I know, the Russian language is affixed to the repository for all games, and we will not be able to determine which games are in English without changing the repository database. @spline1986 can you help with this?

**Comment by valerio-bozzolan** [comment]
Yeah maybe it's just easier to introduce an English-supported repository. So we don't touch the source code.

I think anyone could contribute to this, in such way.
